src/components/HomePage.vue

```diff
@@ -53,9 +53,9 @@
       </div>
     </div>
     <div class="card-footer" id="footer">
-      <a>
-        Copyright (c) 2017 Falko - UnB-FGA (GPP/MDS) Grupo 8 Segundo Semestre de 2017
-      </a>
+      <a><h5>Copyright (c) 2017 Falko</h5></a>
+      <div>All Rights Reserved.</div>
+      <a>This product is protected by copyright and distributed under licenses restricting copying, distribution, and decompilation.</a>
```

Comment:
This text really reflects the license that we're using? As a free Software under MIT license it seems a little too restrict.

Reply:
@MartinsLucas I think you're right